osquery/events/darwin/fsevents.cpp

```diff
@@ -31,6 +32,7 @@
 namespace fs = boost::filesystem;
 
 namespace osquery {
+FLAG(bool, enable_file_events, false, "Enables the file_events publisher");
```

Comment:
We should avoid defining two versions of this flag and instead create a shared `.cpp` (or similar) where only one is defined. This will help keep documentation and the default value consistent.

Reply:
That's a great way to avoid inconsistencies between platforms that I didn't think about until now! Thanks for pointing this out, I will definitely pay more attention to this in the future!